packages/components/Menu/src/MenuItemRadio/MenuItemRadio.tsx

```diff
@@ -2,13 +2,12 @@ import type { UseSlots } from '@fluentui-react-native/framework';
 
 import { useMenuItemRadio } from './useMenuItemRadio';
 import { MenuItemCheckbox, menuItemFinalRender } from '../MenuItemCheckbox/MenuItemCheckbox';
-import type { MenuItemCheckboxProps, MenuItemCheckboxType } from '../MenuItemCheckbox/MenuItemCheckbox.types';
-
-export const menuItemRadioName = 'MenuItemRadio';
+import { menuItemRadioName } from '../MenuItemRadio/MenuItemRadio.types';
+import type { MenuItemRadioProps, MenuItemRadioType } from '../MenuItemRadio/MenuItemRadio.types';
 
 export const MenuItemRadio = MenuItemCheckbox.compose({
```

Comment:
I'm pretty sure the exported type for this component will still be MenuItemCheckboxProps-based.

Reply:
Made the change to radio code. 

Tested it on win32 as well. MenuRadio Works as before but types are migrated to MenuItemRadio now.